src/frontend/js/components/DashboardItem/_styles.scss

```diff
@@ -31,22 +34,36 @@
       justify-content: space-between;
       align-items: center;
       padding: 0.5rem 1rem;
-      color: r-color(denim);
     }
   }
 }
 
 .dashboard-sub-course-run-list {
   padding-left: 30px;
 
-  &__item::before {
-    position: absolute;
-    left: 0;
-    content: '';
-    width: 10px;
-    height: 10px;
-    background: blue;
-    border-radius: 50%;
+  &__item {
+    position: relative;
+
+    &::before {
+      position: absolute;
+      left: -20px;
+      top: 20px;
+      content: '';
+      width: 10px;
+      height: 10px;
+      background: r-color(indianred3);
```

Comment:
You should use `r-theme-val` as within site-factory we must be able to use another color.